app/src/main/java/com/example/triptracker/view/map/RecordScreen.kt

```diff
@@ -276,45 +277,43 @@ fun Map(
     if (!viewModel.isInDescription()) {
       StartWindow(viewModel = viewModel, context)
 
-      if (!viewModel.addSpotClicked.value) {
```

Comment:
why did you remove this check? not saying that it's wrong but I don't understand why it's not present anymore

Reply:
because it was precised on the figma that we can have the center location button